### DISCUSSION TITLE
How to bind <XF86Lauch*> keys to a userscript?

### DISCUSSION THREAD

**Discussion by meissnol** [qutebrowser/qutebrowser#5610] [Q&A] [answered]
I want to bind `<XF86LaunchF>` key in config.py, but can't figure out how the Syntax is. Everything I tried, qb complains about an invalid key.

---

*edited by @The-Compiler to add backticks around the key, so this isn't interpreted as HTML (and hidden) by GitHub*

**Comment by The-Compiler** [comment, marked as answer]
What does `:debug-keytester` say if you press that key?

**Reply by meissnol** [reply]
ehm,... it tells me the correct answer... <Launch (H)> :D

thanks a lot

**Reply by The-Compiler** [reply]
Huh, that's a weird one! It looks like in Qt, `Launch (0)` and `Launch (1)` correspond to "My Computer" and "Calculator" keys, so everything is shifted by 2. Apparently that's there for legacy reasons and (hopefully) going to be fixed in Qt 6.
